components/nds-stacked-nav/src/StackedNav.js

```diff
@@ -58,7 +59,7 @@ export const StackedNavLink = props => {
 		linkProps.to = destination;
 	}
 	return (
-		<li className="stacked-nav__list-item">
+		<li className="stacked-nav__list-item" {...rest}>
```

Comment:
Did we decide what we're doing with merging a className prop?

Reply:
good call - will sort this